src/pages/en/guides/integrations-guide.md

```diff
@@ -13,22 +11,21 @@ i18nReady: true
 - Add new features to your project, like automatic sitemap generation.
 - Write custom code that hooks into the build process, dev server, and more.
 
-> Integrations are still new, and the API has not yet been finalized. Only official Astro integrations (those published to `@astrojs/` on npm) are currently supported to protect users from breaking changes.
->
-> **To enable 3rd-party integrations:** Run Astro with the `--experimental-integrations` CLI flag, or include `experimental: { integrations: true }` in your Astro config file. 
+:::note[Experimental status]
+Integrations are still new, and the API has not yet been finalized. Only official Astro integrations (those published to `@astrojs/` on npm) are currently supported to protect users from breaking changes.
```

Comment:
```suggestion
Only official Astro integrations (those published to `@astrojs/` on npm) are currently supported to protect users from breaking changes.
```

Reply:
One small tweak here just to clarify that it’s a default not zero support. (Next paragraph does imply this but could be confusing reading these side by side.)

```suggestion
Only official Astro integrations (those published to `@astrojs/` on npm) are currently supported by default to protect users from breaking changes.
```